lib/howitzer/web/page.rb

```diff
@@ -28,7 +28,7 @@ class Page
 
       def self.inherited(subclass)
         subclass.class_eval { include Singleton }
-        PageValidator.pages << subclass
+        PageValidator.pages << subclass if subclass.validations.present?
```

Comment:
technically, current implementation work next way... Assume there are 3 pages on site. We have described 2 pages only and 1 abstract page

RealPage1  < AbstractPage, RealPage2 < AbstractPage

Then howitzer raises error for pages without validations in case:
1) RealPage.given, AbstractPage.given
2) RealPage.open, AbstractPage.open
3) RealPage.opened?, AbstractPage.openend?
4) RealPage.displayed?, AbstractPage.dispalyed?

If current page is unknown, or unexpected, then we exclude all abstract pages there

Is it desired implementation?